lang/cs/Org.Apache.REEF.Utilities/Org.Apache.Reef.Utilities.DotNet.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>Org.Apache.Reef.Utilities</AssemblyName>
+    <OutputPath>..\bin\$(AssemblyName)</OutputPath>
+    <TargetFrameworks>netstandard2.0;net46;net451</TargetFrameworks>
+    <NetStandardImplicitPackageVersion Condition=" '$(TargetFramework)' == 'netstandard2.0' ">2.0.0-beta-25123-01</NetStandardImplicitPackageVersion>
+    <DefaultItemExcludes>$(DefaultItemExcludes);**/AssemblyInfo.cs</DefaultItemExcludes>
```

Comment:
Its excluded because dotnet already imports those properties by default and it was causing conflicts. We may have to break out the InternalVisibleTo properties into a different file, but Ill wait to address this when I get to that point.